### ISSUE TITLE
broken code

### ISSUE THREAD

**Issue by linuxbioinfoguy** [nghiavtr/FuSeq#13]
Hi, when executing, it broken and show the message as follow:

 Checking possible paralogs...
 Post processing mapped reads (MR)...Error in 1:nrow(junctBr$myFusionFinal) : argument of length 0
Calls: postProcessMappedRead -> refineJunctionBreak
Execution halted

But I could not find any line with the pattern "1:nrow(junctBr$myFusionFinal)" in the R code.

Thanks

J

**Comment by nghiavtr** [comment]
Hi,

Sorry for this bug.
The error should be from function refineJunctionBreak() called from postProcessMappedRead ().
The R code of this function can be found in FuSeq_functions.R
I am not sure what happened but it is likely that in your data there are no fusion candidates discovered from the mapped-read pipeline.

Best,
Nghia

**Comment by linuxbioinfoguy** [comment]
Thanks, let's see if I could fix that, the reason is that even there is no fusion detected, it probably better to keep the script unbroken and generate a report even it is empty.

Thanks

J

**Comment by nghiavtr** [comment]
Right, it should run smoothly. I will try to fix this in the next version of FuSeq.
Thank you for reporting this bug.

Best,
Nghia
